feature/s3/manager/bucket_region.go

```diff
@@ -25,7 +25,7 @@ const bucketRegionHeader = "X-Amz-Bucket-Region"
 // For example to get the region of a bucket which exists in "eu-central-1"
 // you could provide a region hint of "us-west-2".
 //
-//	cfg, err := config.LoadDefaultConfig(context.TODO())
+//	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion("us-west-2"), config.WithCredentialsProvider(aws.AnonymousCredentials{}))
```

Comment:
How about this?
```suggestion
//	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion("us-west-2"))
```

Reply:
got it
https://github.com/aws/aws-sdk-go-v2/pull/2716#issuecomment-2233770483

Will remove the blurb about not using cred as well.